CODEOWNERS

```diff
@@ -1,8 +1,2 @@
 # These owners will be the default owners for everything in the repo.
-*         @esri/calcite-components
-
-# docs:
-/docs/**  @esri/all-esri-members
-
-# Template and assets
-*.md      @esri/all-esri-members
+*     @esri/calcite-components-reviewers
```

Comment:
@julio8a @jcfranco  can we rename this group to `calcite-codeowners`?

Reply:
I'm fine either way. Something that applies to reviews and code ownership works for me.